docker-compose.yml

```diff
@@ -46,6 +56,9 @@ services:
       INFLUXURL: http://influxdb:8086
       NUOCMD_API_SERVER: nuoadmin1:8888
       NUOCD_HOSTNAME: sm
+      INFLUXDB_TOKEN: $DOCKER_INFLUXDB_INIT_ADMIN_TOKEN
+      INFLUXDB_BUCKET: $DOCKER_INFLUXDB_INIT_BUCKET
+      INFLUXDB_ORG: $DOCKER_INFLUXDB_INIT_ORG
```

Comment:
We need to document the new environment variables in the [README.md](https://github.com/nuodb/nuodb-collector#running-nuodb-collector).